nextjs-app/src/pages/index.tsx

```diff
@@ -7,20 +7,22 @@ import { PieSpinner } from "@justeattakeaway/pie-spinner/dist/react"
 import { PieModal } from "@justeattakeaway/pie-modal/dist/react"
 import { PieCookieBanner } from "@justeattakeaway/pie-cookie-banner/dist/react"
 import { PieIconButton } from "@justeattakeaway/pie-icon-button/dist/react"
-import { IconClose } from "@justeattakeaway/pie-icons-webc/dist/react/IconClose";
+import { IconClose } from "@justeattakeaway/pie-icons-webc/dist/react/IconClose"
```

Comment:
Doesn't block your PR but why isn't linting requiring semi colons here?

Reply:
I'm not entirely sure but from looking at the eslint setting we dont have a rule for semi colons. I added one in locally and now it errors.